<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutForm</class>
 <widget class="QDialog" name="AboutForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>706</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>740</width>
    <height>706</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>740</width>
    <height>706</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Помощь</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/question.png</normaloff>:/icons/icons/question.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget
{
	background-color: #212630;
	color: #CFD2DC;
	border-color: #1C2227;

}

QFrame
{
	background-color: #212630;
	border: 1px solid #212630;
	border-radius: 10px;
}


/*-----QComboBox-----*/
QComboBox
{
    background-color: #232935;
    border: 2px solid;
    border-radius: 15px;
	border-color: #3BB0D7;
    padding-left: 6px;
    color: #CFD2DC;
    height: 20px;

}


QComboBox::disabled
{
	background-color: #232935;
	color: #CFD2DC;
	border-color: #051a39;

}


QComboBox:hover
{
    background-color: #353C4F;

}


QComboBox:on
{
    background-color: #3BB0D7;

}


QComboBox QAbstractItemView
{
    background-color: #1C2227;
    color: #CFD2DC;
    border: 1px solid black;
    selection-background-color: #3BB0D7;
    selection-color: #CFD2DC;
    outline: 0;

}


QComboBox::drop-down
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left-width: 0px;
    border-left-color: darkgray;
    border-left-style: solid; 
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px;

}


QComboBox::down-arrow
{
    image: url(:/icons/icons/arrow-down.png);
    width: 8px;
    height: 8px;
}

/*-----QPushButton-----*/
QPushButton
{
	background-color: #232935;
	color: #CFD2DC;
	font-weight: bold;
	border-style: solid;
	border-width: 2px;
	border-radius: 30px;
	border-color: #3BB0D7;
	padding: 10px;
	highlited:true;
}


QPushButton::hover
{
	background-color: #353C4F;
	color: #CFD2DC;

}


QPushButton::pressed
{
	background-color: #3BB0D7;
	color: #CFD2DC;

}

/*-----QSpinBox &amp; QDoubleSpinBox &amp; QDateTimeEdit-----*/
QSpinBox, 
QDoubleSpinBox,
QDateTimeEdit
{
	background-color: #232935;
	color: #CFD2DC;
	border: 2px solid #3BB0D7;
	border-radius: 10px;
	padding : 2px;

}



QSpinBox:hover, 
QDoubleSpinBox::hover,
QDateTimeEdit::hover
{
    background-color: #232935;
    border: 2px solid #6CB2FF;
    color:  #CFD2DC;
    padding: 2px

}


QSpinBox::up-button,
QDoubleSpinBox::up-button,
QDateTimeEdit::up-button
{
    background-color: #3BB0D7;
	border-top-right-radius: 5px;
	border-top-left-radius: 1px;

}

QSpinBox::down-button,
QDoubleSpinBox::down-button,
QDateTimeEdit::down-button
{
    background-color: #3BB0D7;
	border-bottom-right-radius: 5px;
	border-bottom-left-radius: 1px;

}

QSpinBox::up-button:hover, QSpinBox::down-button:hover,
QDoubleSpinBox::up-button:hover, QDoubleSpinBox::down-button:hover,
QDateTimeEdit::up-button:hover, QDateTimeEdit::down-button:hover
{
    background-color: #45CDFA;
    border: 2px solid #45CDFA;

}

QSpinBox::up-button:pressed, QSpinBox::down-button:pressed,
QDoubleSpinBox::up-button:pressed, QDoubleSpinBox::down-button::pressed,
QDateTimeEdit::up-button:pressed, QDateTimeEdit::down-button::pressed
{
    background-color: #CFD2DC;
    border: 2px solid #CFD2DC;

}

QSpinBox::down-arrow,
QDoubleSpinBox::down-arrow,
QDateTimeEdit::down-arrow

{
    image: url(:/icons/icons/arrow-down.png);
    width: 7px;

}


QSpinBox::up-arrow,
QDoubleSpinBox::up-arrow,
QDateTimeEdit::up-arrow

{
    image: url(:/icons/icons/arrow-up.png);
    width: 7px;

}


</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>720</width>
     <height>696</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
       </font>
      </property>
      <property name="text">
       <string>&lt;h2&gt;Помощь&lt;/h2&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
         </font>
        </property>
        <property name="text">
         <string>&lt;h3&gt;Условия вклада&lt;/h3&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_term_1">
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Условия 1: Начисление процентов каждый год. 
Минимальный срок вклада: 3 года.
Процентная ставка: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_term_2">
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Условия 2: Начисление процентов каждый месяц. 
Минимальный срок вклада: 1 год.
Процентная ставка: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_term_3">
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Условия 3: Начисление процентов ежедневно 
на каждую полную тысячу вклада. 
Минимальный срок вклада: 6 месяцев.
Процентная ставка: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_term_4">
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Условия 4: Начисление процентов ежедневно 
на полную сумму вклада. 
Минимальный срок вклада: 3 месяца.
Процентная ставка: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_max_2">
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Максимальный срок вклада для всех условий: 10 лет.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_min">
          <property name="font">
           <font>
            <family>Segoe UI Semibold</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Минимальная сумма вклада: 10000 RUB / 150 USD / 150 EUR</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_max">
          <property name="font">
           <font>
            <family>Segoe UI Semibold</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Максимальная сумма вклада: 100000000 RUB / 1500000 USD / 1500000 EUR</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
